doc/parameters.md

```diff
@@ -4,124 +4,134 @@ APIcast v2 has a number of parameters configured as environment variables that c
 
 Note that when deploying APIcast v2 with OpenShift, some of thee parameters can be configured via OpenShift template parameters. The latter can be consulted directly in the [template](https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml).
 
-- `APICAST_CUSTOM_CONFIG`
+#### `APICAST_CUSTOM_CONFIG`
```

Comment:
The only reason is that it would be huge with `##`...

![screen shot 2017-03-13 at 17 10 48](https://cloud.githubusercontent.com/assets/1270328/23863438/12da67d2-0810-11e7-9e04-b82b39dedaea.png)

I wanted to find a way to not use headers, and maybe just manually add anchors, but couldn't make it work. So, `####` was an easy and visually nice compromise.